roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -201,7 +213,11 @@ class Namespace(object):
 
 
 class ResetNeutron(object):
-    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-proxy', 'neutron-linuxbridge-agent']
+    all_agents = ['neutron-dhcp-agent', 'neutron-l3-agent', 'neutron-metadata-agent', 'neutron-linuxbridge-agent']
```

Comment:
Let's reformat this onto multiple lines so it's easier to tell when agents are renamed.